src/main/java/spoon/reflect/declaration/CtPackage.java

```diff
@@ -161,4 +161,10 @@ public interface CtPackage extends CtNamedElement, CtShadowable {
 	 * @return true if the package contains no types nor any other packages
 	 */
 	boolean isEmpty();
+
+	/**
+	 * @return true if the package contains any types.
+	 * @see #getTypes()
+	 */
```

Comment:
Now that I think about it, we should probably add a requirement here that this is O(1). That's really the reason for it to exist in the first place.

On that note, we maybe should add notes to `getTypes()` and `getPackages()` that they may be linear in the amount of types/packages, and one should use `isEmpty()`, `hasPackages()`  (if we add it) or `hasTypes()` to check for "emptiness".